### PR TITLE
Upload Pipeline: test older containers

### PR DESCRIPTION
**Pull request by sfraint**
* Test all containers from the past ~3 months
* Test at least `n` Batfish releases and `m` Pybatfish releases, regardless of their age

### CODE REVIEW

**Review by sfraint — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/batfish/docker/93)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @arifogel, @dhalperi, and @sfraint)

___
*[`.buildkite/pipeline.sh` line 257 at r1](https://reviewable.io/reviews/batfish/docker/93#-NA61XxJBqK81J9BSWUA:-NA61XxJBqK81J9BSWUB:b-pa8575) ([raw file](https://github.com/batfish/docker/blob/598c3bcf1e7c4ed1dad15f590b5a79414770a665/.buildkite/pipeline.sh#L257)):*
> ```Shell
> # Get available Pybatfish versions from PyPI
> python -m pip install --user 'requests==2.23.0' >/dev/null
> PYBF_TAGS=$(python -c "import requests; print('\n'.join(requests.get('https://pypi.python.org/pypi/pybatfish/json').json()['releases'].keys()))" | sort -r)
> ```

This isn't right for pybf yet, since months and days are canonicalized to drop leading zeros



_Code quote:_
```Shell
| sort -r
```

**Review by sfraint — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/batfish/docker/93)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @arifogel and @dhalperi)

___
*[`.buildkite/pipeline.sh` line 257 at r1](https://reviewable.io/reviews/batfish/docker/93#-NA61XxJBqK81J9BSWUA:-NA66rdcE0vFY6E6_xbi:b-n6jbzs) ([raw file](https://github.com/batfish/docker/blob/598c3bcf1e7c4ed1dad15f590b5a79414770a665/.buildkite/pipeline.sh#L257)):*
```
$ python -c "
> from datetime import datetime
> import requests
> releases = requests.get('https://pypi.python.org/pypi/pybatfish/json').json()['releases'].keys()
> print('\n'.join(sorted(releases, key=lambda x: datetime.strptime(x, '%Y.%m.%d.%f'), reverse=True)))
> "
2022.3.16.1242
2022.2.18.1216
2021.11.4.1095
2021.7.9.974
2021.4.12.882
2021.2.26.830
2020.12.23.763
2020.10.8.667
2020.8.11.600
2020.4.23.480
2020.2.20.414
2020.2.3.395
2020.1.11.363
2019.11.20.299
2019.11.5.282
2019.11.1.277
2019.11.1.276
```

**Review by dhalperi — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/batfish/docker/93)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @arifogel and @sfraint)

___
*[`.buildkite/pipeline.sh` line 27 at r2](https://reviewable.io/reviews/batfish/docker/93#-NA67UdW9irpqrfBXrFX:-NA67UdW9irpqrfBXrFY:b-o2tpsp) ([raw file](https://github.com/batfish/docker/blob/423364d179b18ab34bc16a182637b00f3c7d2a52/.buildkite/pipeline.sh#L27)):*
> ```Shell
> BATFISH_VERSION_STRING="${BATFISH_VERSION_STRING:-$(date +'%Y.%m.%d').${BUILDKITE_BUILD_NUMBER}}"
> 
> # Only test Bf containers less than this many days old
> ```

I'd reword this to more accurately represent what's up?



_Code quote:_
```Shell
Only test Bf containers less than this many days old
```

**Review by sfraint — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/batfish/docker/93)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @arifogel and @dhalperi)

___
*[`.buildkite/pipeline.sh` line 27 at r2](https://reviewable.io/reviews/batfish/docker/93#-NA67UdW9irpqrfBXrFX:-NA6F9q99S9tFHkWclmn:bvnrpcm) ([raw file](https://github.com/batfish/docker/blob/423364d179b18ab34bc16a182637b00f3c7d2a52/.buildkite/pipeline.sh#L27)):*
<details><summary><i>Previously, dhalperi (Dan Halperin) wrote…</i></summary><blockquote>

I'd reword this to more accurately represent what's up?
</blockquote></details>

updated

**Review by dhalperi — COMMENTED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/batfish/docker/93)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @arifogel)
